exonum/src/api/private/system.rs

```diff
@@ -148,6 +147,16 @@ impl SystemApi {
         self.info.clone()
     }
 
+    fn get_consensus_enabled_info(&self) -> bool {
+        self.shared_api_state.is_enabled()
+    }
```

Comment:
I think the naming conventions for getter/setters is `foo/set_foo`? Also, the `info` word does not any information, so I suggest renaming to `consensus_enabled/set_consensus_enabled`.

Reply:
I've used `get_` and `_info` here to match the surrounding code. This naming convention is used in the current file and in [`public/system.rs`](https://github.com/exonum/exonum/blob/0ff5c1a81/exonum/src/api/public/system.rs#L71-L99)

Reply:
:( 

We should really rename those as well...

Reply:
I also strongly dislike such naming. This function should be named `is_consensus_enabled`.

I understand your concern about naming consistency, although prefer to make at least some improvements, even if that leads to inconsistency. All these things can be renamed in a separate pull request if you insist. 😄

Reply:
I'll create another issue+PR for this.